.github/workflows/e2e.yml

```diff
@@ -67,9 +66,9 @@ jobs:
           kubectl create ns kyma-system || true
           make deploy IMG=$MANAGER_IMAGE
 
-      - name: Setup NATS
+      - name: Setup NATS-Manager
         run: |
-          make -C hack/ci/ install-nats-module
+          make -C hack/ci/ install-nats-manager
```

Comment:
Can we keep it as `install-nats-module`? as its still a module, IMO. For now, lets keep it for now and change it after team agreement.

Reply:
reverted!